great-idea/services.html

```diff
@@ -1,87 +1,123 @@
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
 
-Learn More
-
-
-
-UX / UI
-
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
-
-Learn More
+  <!--[if lt IE 9]>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+  <![endif]-->
+</head>
 
+  <body>
 
+<!-- copy and paste your index.html page structure in here, markup the content provided after that -->
+    <div class="container">
+
+        <header>
+            <nav>
+              <a href="services.html">Services</a>
+              <a href="#">Product</a>
+              <a href="#">Vision</a>
+              <a href="#">Features</a>
+              <a href="#">About</a>
+              <a href="#">Contact</a>
+            </nav>
+        
+            <a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+          </header>
 
-Digital Marketing
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+        <section class="services-intro">
+            <h4>Services</h4>
 
-Learn More
+            <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+        </section>
 
+        <section class="services-main-content">
+            <div class="digital-design services-content-box">
+                <h5>Digital Design</h5>
 
+                <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                eleifend. Mauris euismod facilisis iaculis.</p>
 
-Web Development
+                <button>Learn More</button>
```

Comment:
Some additional styling you can add to the buttons are border-radius and font-size. I would make a class specific for the button's style so it will affect all of them. Also, you can have more than one button styling class for different types of buttons.